service/frontend/operatorHandler.go

```diff
@@ -0,0 +1,303 @@
+// The MIT License
```

Comment:
Go idiomatic name for this file is `operator_handler.go`. I know, we didn't follow this rule, but all new stuff we ads is following. One day I will rename other files.